app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,108 @@
+class SubmissionsController < ApplicationController
+
+  def new
+    @project = Project.find(params[:project_id])
+    @submission = @project.submissions.build
```

Comment:
Does this work:

```ruby
@submission = @project.submissions.build(
  file: @project.directory,
  cores: '28',
  extra: '-verb',
  email: false,
  scheduled_hrs: 1
)
```

Also there are other ways to provide default values to models besides setting them in the controller. Example: custom getter that returns `value || default`, or using after_initialize callback to set default values.

Finally, why is cores a string instead of an integer?

Reply:
Note: those last two comments are more of an FYI about Rails and just curiosity, I am not concerned about types (int or string) at this stage.